### PR TITLE
Refactor: change the last open statement to context manager

### PR DESCRIPTION
**Pull request by divyenduz**

### CODE REVIEW

File: db_s3_backup.py [hunk 1]
```diff
@@ -177,16 +177,16 @@ def delete_local_backups(dir_path, backup_prefix, backup_extension, verbose = Fa
 
 	# Loads configuration file from JSON
 	try:
-		f=open(args.config_file, 'r')
+		with open(args.config_file, 'r') as f:
+			try:
+				config=json.loads(f.read())
+			except Exception, e:
```
Comment (mittsh) [inline db_s3_backup.py]: Can you change this to `ValueError`? It's the right error for JSON parsing error.

File: db_s3_backup.py [hunk 2]
```diff
@@ -177,16 +177,16 @@ def delete_local_backups(dir_path, backup_prefix, backup_extension, verbose = Fa
...
+				print(e)
+				print('Cannot parse configuration file (must be JSON).')
+				exit(1)
 	except Exception, e:
```
Comment (mittsh) [inline db_s3_backup.py]: Can be changed to `IOError`

**Review by mittsh — CHANGES_REQUESTED** [review]
Taking the chance to replace the general `Exception` by specific ones.

**Comment by mittsh** [comment]
Thanks @divyenduz ! Ideally we could take this as a change to fine tune the exception handling…

**Comment by divyenduz** [comment]
@mittsh Yes, that will make the code better. The changes have been made :)

**Review by mittsh — APPROVED** [review]

**Comment by mittsh** [comment]
Thanks @divyenduz looks great

**Pull request merged**